jsonmerge_utils.py

```diff
@@ -51,14 +72,14 @@ def main():
     """
     try:
         configure_logger()
-        data_dir = input('Folder Path: ')
+        data_dir = args.filepath

```

Comment:
dont need this variable anymore, we can directly pass _args.filepath_ wherever needed